templates/core/terraform/locals.tf

```diff
@@ -21,3 +21,10 @@ data "http" "myip" {
 locals {
   myip = var.public_deployment_ip_address != "" ? var.public_deployment_ip_address : chomp(data.http.myip[0].body)
 }
+
+locals {
+  tre_core_tags = {
+    tre_id              = var.tre_id
+    tre_core_service_id = var.tre_id
```

Comment:
is this value correct?

Reply:
yes, by design.
we need it to descriminate core service from other services in a TRE instance

Reply:
discussted over teams:

when you call Azure Cost Management REST API and group by tag and filter by tre_id tag you get in response aggregated costs by tag.

due to the fact that we don't have a generated id per core service, I chose to give it the same id as the tre id